changelogs/8.13.asciidoc

```diff
@@ -0,0 +1,31 @@
+[[release-notes-8.13]]
+== APM version 8.13
+
+https://github.com/elastic/apm-server/compare/8.12\...8.13[View commits]
```

Comment:
Side note: @bmorelli25 fixed all the other branches to not produce this branch diff, and instead produce diffs between patches. We may have to update automation to be consistent.